pages/linux/wine.md

```diff
@@ -0,0 +1,32 @@
+# wine
+
+> Run Windows programs on Unix.
+> More information: <https://wiki.winehq.org/List_of_Commands>.
```

Comment:
Is this the best link for us to use? 
It doesn't seem to be directly talkin about Wine. It actually talks about wine subcommands. 
Maybe we can use the official documentation 
```suggestion
> More information: <https://www.winehq.org/documentation>.
```

or the main page

```suggestion
> More information: <https://www.winehq.org/>.
```

Reply:
I agree. the main page is more appropriate reference for help with the tool. I will update the page.